game/resources.cpp

```diff
@@ -102,9 +103,23 @@ Resources::Resources(Tempest::Device &device)
   Pixmap pm(1,1,Pixmap::Format::RGBA);
   fbZero = device.texture(pm);
   }
+  }
 
+void Resources::loadAssets(const std::vector<std::u16string>& modvdfs) {
   std::vector<Archive> archives;
-  detectVdf(archives,Gothic::inst().nestedPath({u"Data"},Dir::FT_Dir));
+  inst->detectVdf(archives,Gothic::inst().nestedPath({u"Data"},Dir::FT_Dir));
+
+  // Remove all mod files, that are not listed in modvdfs
+  archives.erase(std::remove_if(archives.begin(), archives.end(),
```

Comment:
Nitpick: `std::erase_if(archives, ..`

Reply:
`std::erase_if` is C++20 (https://en.cppreference.com/w/cpp/container/vector/erase2), you currently have C++17 set in CMakeLists. Do you want to update this now?

Reply:
Right, we need to update first. So, let's just leave `remove_if` for now then